pom.xml

```diff
@@ -226,6 +230,11 @@ file comparators, endian transformation classes, and much more.
   </contributors>
 
   <dependencies>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
```

Comment:
Is this just to bring in the @javax.annotation.concurrent.ThreadSafe annotation? I don't think this necessary since you already document the thread safety in the javadoc. Nothing else in the code base uses this non-standard annotation support.

Reply:
Well yes it is to bring in the annotation... but these annotations can also be used by various tools like FindBugs and others to detect issues.

I think it adds value, and jsr305 is tiny (19KB). I can remove it if people prefer? It also adds useful annotations like `@Nullable` and `@NotNull` for similar purposes.

Reply:
I would remove it from this PR. Adding an annotation dependency should be in another PR that perhaps then makes more use of it than a single location. The Guava codebase uses these throughout and in that case they have value for a bug detection tool. I don't think a single annotation of `@ThreadSafe` is likely to detect many (any?) bugs given that synchronized code should be less of a problem than unsynchronized code. If you do have a bug detection tool that flags this class as a problem when used in a concurrent scenario a brief look at the javadoc (which notes this is safe for concurrent use) will allow you to mark the bug as false positive.

Reply:
Okie dokie. Done.